packages/suite/src/middlewares/wallet/coinjoinMiddleware.ts

```diff
@@ -276,14 +274,14 @@ export const coinjoinMiddleware =
 
         if (action.type === COINJOIN.CLIENT_SESSION_PHASE) {
             const { accountKeys } = action.payload;
-            const isAlredyInterrupted = api
+            const isAlredyPaused = api
```

Comment:
Please fix the typo here: `isAlreadyPaused`

Reply:
done